.github/workflows/octave_moxunit_tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         with:
           file: ./coverage.xml
           verbose: true
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
Maybe we can avoid all CI pipeline because of coverage, is the wrong signal if the coverage decreases 0.01% to see the pipeline red IMHO, WDYT ?

Reply:
I think that these are not related.
1. If the coverage decreases, github shows (and it should) a red cross.
2. the increase of code coverage is not mandatory for each PR 
3. Finally, in moxunit, the parameter you propose changing states that the CI fails if errors are found in the execution, which I think it is ok, thus I suggest leaving this like this.

Reply:
Ok, will revert the changes.

Reply:
Done